source/apps/metadata.txt

```diff
@@ -0,0 +1,250 @@
+.. _app-metadata:
+
+================
+Get App Metadata
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _find-atlas-project-id:
+
+Find Your Atlas Project ID
+--------------------------
+
+Every App is associated with a single Atlas project. You use the
+project's unique Project ID value, sometimes referred to as a "Group
+ID", to identify and work with your App programatically.
+
+You can find your Project ID in the Atlas UI or use the Atlas Admin API
+or Atlas CLI to find it programmatically.
+
+.. tabs::
+
+   .. tab:: Atlas UI
+      :tabid: ui
+
+      You can find your Project ID in the App Services Admin UI.
+
+      .. procedure::
+
+         .. step:: Choose Your Atlas Organization
+
+            In the upper left-hand corner of your Atlas Dashboard, you'll see
+            a drop-down menu that contains the Organizations you can access.
+            Select the Organization that contains the Project you'd like to
+            view.
+
+            .. figure:: /images/select-an-organization.png
+               :alt: Menu showing a list of available Organizations
+               :width: 250px
+
+         .. step:: Find Your Project
+
+            Click :guilabel:`Projects` in the left navigation menu to view the
+            list of project in the Organization. Find your project in the
+            list.
+
+            .. figure:: /images/view-the-projects-list.png
+               :alt: The left navigation menu with the project selection highlighted
+               :width: 250px
+
+         .. step:: Copy the Project ID
+
+            The right side of the project list view contains an :guilabel:`Actions`
+            column. Click the ellipses icon to expand the Actions menu, and select
```

Comment:
```suggestion
            column. Click the ellipses (:guilabel:`...`) icon to expand the Actions menu, and select
```

nit: don't think ellipses is nec a known word, esp for non-native devs

Reply:
also i think cloud/server represents stuff like this w icons. could be worth lukin into